src/aur.c

```diff
@@ -385,6 +404,28 @@ static int json_integer (void * ctx, long long val)
 	return 1;
 }
 
+static int json_double (void * ctx, double val)
```

Comment:
I agree, I removed non string attributes (larchunix/package-query@fa823c2) and didn't see any regression.